portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -56,12 +59,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     long timestamp = System.currentTimeMillis();
-    String comment = request.getParameter("text-input");
+    String comment = request.getParameter("message");
+    String author = request.getParameter("name");
     response.setContentType("text/html");
 
```

Comment:
Maybe this can come later as part of https://github.com/googleinterns/step/blob/master/walkthroughs/week-3-server/comments-walkthrough.md#next-steps, but I would prevent empty user/comments from going through.